index.js

```diff
@@ -228,7 +228,7 @@ function createPagination(data, req, limit){
 	pagination.page = pagination.page > pagination.pages ? pagination.pages : pagination.page 
 	if ( pagination.page + 1 <= pagination.pages   ) pagination.pageNext = pagination.page + 1 
 	if ( pagination.page - 1 > -1                  ) pagination.pagePrev = pagination.page - 1 
-	pagination.data = data.splice( pagination.page * limit, limit )
+	pagination.data = data.splice( (pagination.page-1) * limit, limit )
```

Comment:
This change actually seems to cause an error for me where page 0 is the last elements because the splice start index becomes negative and it starts from the end [see the docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice#Parameters).

Also, we're taking the result of the splice which is the remove elements so the current code where we don't subtract one seems right to me. Can you double check this change?